packages/autocomplete-core/src/__tests__/getInputProps.test.ts

```diff
@@ -1905,6 +1905,8 @@ describe('getInputProps', () => {
       inputElement.focus();
       inputElement.blur();
 
+      await runAllMicroTasks();
```

Comment:
Hard to say since tests are passing, but new features or bug fixes might "break" tests that actually just need to wait.

We should tackle this, but likely in a separate PR.